examples/flink-consumer-source/src/main/java/org/apache/flink/streaming/connectors/pulsar/example/README.md

```diff
@@ -95,3 +95,186 @@ $ bin/pulsar-client consume -n 0 -s test test_dest
 ```
 
 You will see similar results as what you see at step 6 when running the word count example to print results to stdout.
+
+
+### PulsarConsumerSourceWordCountToAvroTableSink
+
+This Flink streaming job is consuming from a Pulsar topic and counting the wordcount in a streaming fashion. The job can write the word count results
+to csv file or another Pulsar topic for json format.
+
+The steps to run the example:
+
+1. Start Pulsar Standalone.
```

Comment:
`PulsarConsumerSourceWordCount` already explains `Step 1, 2 and 3` so we can define as `Step 1, 2 and 3 are same as above.` in 
```
PulsarConsumerSourceWordCountToAvroTableSink
PulsarConsumerSourceWordCountToJsonTableSink
```
Ref: https://github.com/apache/pulsar/blob/master/examples/flink-consumer-source/src/main/java/org/apache/flink/batch/connectors/pulsar/example/README.md